src/module/hotKeysModule.ts

```diff
@@ -515,9 +568,30 @@ export class HotKeysModule {
         });
     }
 
-    resetStash() {
-        this.lastStash = null;
-        this.nextStash = null;
+    connectTileableFocusChange() {
+        this.tilableFocusChangeConnectionId = Me.msWorkspaceManager
```

Comment:
I probably could connect to 'focus-changed' event from `Me.msWindowManager.msFocusManager` instead, which would rather made the code simpler, but this event is not emitted for the MsApplicationLauncher and `tileable-focus-changed` is. Is it a bug?